docs/source/admin/fs/ssh.rst

```diff
@@ -93,3 +93,18 @@ To specify the drive, you need to use the following format:
      username: "username"
      password: "password"
      protocol: "ssh"
+
+Windows shared folders
```

Comment:
Is that implemented?

Reply:
Yes. It's already implemented. We were using this way when SMB crawler is not done. And if user has server ssh access, this way is more reliable since there are three versions about SMB. It's not easy to maintain for us but the OS has already done it.

Reply:
I'll do some tests then explain that in the doc.